kafka/consumer/group.py

```diff
@@ -552,11 +552,8 @@ def committed(self, partition):
                 committed = None
         return committed
 
-    def topics(self):
-        """Get all topics the user is authorized to view.
-
-        Returns:
-            set: topics
+    def _fetch_all_topic_metadata(self):
+        """A blocking call that fetches topic metadata for all topics in the cluster.
```

Comment:
I'm guessing the statement `...that the user is authorized to view.` also applies here?